crates/svm-rs/src/install.rs

```diff
@@ -159,17 +159,21 @@ struct Installer<'a> {
 impl Installer<'_> {
     /// Installs the solc version at the version specific destination and returns the path to the installed solc file.
     fn install(self) -> Result<PathBuf, SvmError> {
-        let solc_path = version_binary(&self.version.to_string());
+        let named_temp_file = tempfile::NamedTempFile::new_in(data_dir())?;
+        let (mut f, temp_path) = named_temp_file.into_parts();
```

Comment:
ah okay, all this does is write to a new tmp file and then move it to the correct location?